docs/development/release_management.rst

```diff
@@ -22,59 +22,48 @@ Pre-Release
                 passing as well as at least one approving review prior
                 to merging into the release branch.
 
-#. For each release candidate, update the version and
-   changelog. Collect a list of the important changes in the release,
-   including their GitHub issues or PR numbers, then run the
-   ``update_version.sh`` script, passing it the new version in the
-   format ``major.minor.patch~rcN``, e.g.::
-
-     securedrop/bin/dev-shell ../update_version.sh 1.3.0~rc1
-
-   The script will open both the main repository changelog
-   (``changelog.md``) and the one used for Debian packaging in an
-   editor, giving you a chance to add the changes you collected. In
-   the Debian changelog, we typically just refer the reader to the
-   ``changelog.md`` file.
-
-#. If you would like to sign the release commit, you will need to do so manually:
-
-   a. Create a new signed commit and verify the signature::
-
-        git reset HEAD~1
-        git commit -aS
-        git log --show-signature
-
-   #. Ensure the new commit is signed, take note of the commit hash.
-
-   #. Edit ``1.x.y-rcN.tag`` and replace the commit hash with the new
-      (signed) commit hash.
+#. For each release candidate, update the version and changelog.
 
-   #. Delete the old tag and create a new one based on the tag file
-      edited above::
+   a. Collect a list of important changes in the release (see 
+      https://github.com/freedomofpress/securedrop/milestones) including GitHub issues or PR 
+      numbers for each change. You will add these changes to the changelog in the next step.
 
-         git tag -d 1.x.y-rcN
-         git mktag < 1.x.y-rcN.tag > .git/refs/tags/1.x.y-rcN
+   b. Run ``update_version.sh``, passing it the new version in the format 
+      ``<major>.<minor>.<patch>~rcN`` (note how we use a tilde here instead of a dash). The script 
+      will open both the main repository changelog (``changelog.md``) and the one used for Debian 
+      packaging in an editor, giving you a chance to add the changes you collected. In the Debian 
+      changelog, we typically just refer the reader to the ``changelog.md`` file:
+      ``securedrop/bin/dev-shell ../update_version.sh 1.3.0~rc1``
 
-#. Push the branch and tags:
+   c. Sign the commit added by the ``update_version.sh`` script:
+      ``git commit --all --amend --gpg-sign --no-edit``
 
-   * For ``1.x.y~rc1``, push the ``release/1.x.y`` branch and
-     ``1.x.y-rc1`` tag directly.
+   d. Push the branch:
+      ``git push origin release/<major>.<minor>.<patch>``
 
-   * For subsequent release candidates and the final release version,
-     issue a PR with changelog and version changes into the
-     ``release/1.x.y`` branch, and push the signed tag once the PR is
-     merged.
+   e. Push the unsigned tag (you'll sign the <major>.<minor>.<patch> tag before the release goes 
+      out):
+      ``git push origin <major>.<minor>.<patch>-rcN``
```

Comment:
We seem to be a bit hand-wavy about the _why_ of tildes-versus-hyphens in the release candidate version strings. Consider a pop-out "Note" elsewhere in this doc to explain that Debian packages denote rcs with tildes, linking out to e.g. https://www.debian.org/doc/manuals/maint-guide/first.en.html#namever, but git tags don't support tildes, linking out to e.g. https://git-scm.com/docs/git-check-ref-format#_description